open-apparel-registry/open-apparel-registry
# Update copy

## Overview

The info website now contains a comprehensive guide for how to
contribute to the OAR. Consequently, we've shortened the text on the
Contribute page, and linked to the guide instead of replicating the
guide on the Contribute page. We've also removed the CSV contributor 
templates for download, as they are now hosted on the info site.

Updates the text about Cookie Preferences in the popup and in Settings
to link to the Terms of Use and Privacy Policy in the new info site.

Connects #1514, #1513 

## Demo

<img width="1049" alt="ContributeA" src="https://user-images.githubusercontent.com/21046714/139698587-627bf79f-9f9b-4bd3-ae05-466ac3850ea1.png">
<img width="1060" alt="ContributeB" src="https://user-images.githubusercontent.com/21046714/139698602-e3fd6472-c630-45dd-9ab7-5e6be17be65c.png">

<img width="635" alt="CookiesSettings" src="https://user-images.githubusercontent.com/21046714/139698615-6d94979c-2c8a-492a-bedb-6c3ca53feedb.png">
<img width="623" alt="CookiesPopup" src="https://user-images.githubusercontent.com/21046714/139698630-7da8e809-e1aa-4ab1-a984-dc2ea9ebf257.png">

## Testing Instructions

* Run `./scripts/server`
* Confirm the text on the [Contribute Page](http://localhost:6543/contribute) matches the [copy](https://docs.google.com/document/d/1pQVF8yoIAWdP9C4RkmCLSGthy429Sh3s-cpW3vl_np8/edit).
* Confirm the info site links open in new tabs. 
* Confirm the text in [settings](http://localhost:6543/settings) matches the text in #1513. Confirm the links open to the correct pages in new tabs. 
* Click on 'Cookie Preferences' in the footer and confirm the text matches the text in #1513. Confirm the links open to the correct pages in new tabs. 
* Confirm the templates have been removed. 

## Checklist
